test/utils/TestConstantsAPI1812.scala

```diff
@@ -0,0 +1,351 @@
+import play.api.libs.json.{JsObject, JsValue, Json}
+
+object TestConstantsAPI1812 {
+
+
+  val lppWithoutOptionalFields: JsObject = Json.obj(
+    "penaltyNumber" -> "1234ABCD",
+    "penaltyCategory" -> "LPP1",
+    "penaltyStatus" -> "P",
+    "penaltyAmountAccruing" -> 123.45,
+    "penaltyAmountPosted" -> 123.45,
+    "penaltyChargeCreationDate" -> "2022-01-01",
+    "communicationsDate" -> "2022-01-01",
+    "penaltyChargeReference" -> "CHARGE123456",
+    "penaltyChargeDueDate" -> "2022-02-01",
+    "principalChargeDueDate" -> "2022-03-01",
+    "principalChargeReference" -> "CHARGING12345")
+
+
+  val lppWithOptionalFields: JsObject = Json.obj(
+    "penaltyNumber" -> "1234ABCD",
+    "penaltyCategory" -> "LPP1",
+    "penaltyStatus" -> "P",
+    "penaltyAmountAccruing" -> 123.45,
+    "penaltyAmountPosted" -> 123.45,
+    "penaltyChargeCreationDate" -> "2022-01-01",
+    "communicationsDate" -> "2022-01-01",
+    "penaltyChargeReference" -> "CHARGE123456",
+    "penaltyChargeDueDate" -> "2022-02-01",
+    "appealStatus" -> "1",
+    "appealLevel" -> "1",
+    "principalChargeDueDate" -> "2022-03-01",
+    "principalChargeReference" -> "CHARGING12345")
+
+  val lppModelWithOptionalFields: LatePaymentPenalty = LatePaymentPenalty(
+    penaltyNumber = "1234ABCD",
+    penaltyCategory = LPPPenaltyCategoryEnum.firstPenalty,
+    penaltyStatus = LPPPenaltyStatusEnum.Posted,
+    penaltyAmountAccruing = 123.45,
+    penaltyAmountPosted = 123.45,
+    penaltyChargeCreationDate = LocalDate.of(2022, 1, 1),
+    communicationsDate = LocalDate.of(2022, 1, 1),
+    penaltyChargeReference = "CHARGE123456",
+    penaltyChargeDueDate = LocalDate.of(2022, 2, 1),
+    appealStatus = Some("1"),
+    appealLevel = Some("1"),
+    principalChargeReference = "CHARGING12345",
+    principalChargeDueDate = LocalDate.of(2022, 3, 1)
+  )
+
+  val modelWithoutOptionalFields: LatePaymentPenalty = LatePaymentPenalty(
+    penaltyNumber = "1234ABCD",
+    penaltyCategory = LPPPenaltyCategoryEnum.firstPenalty,
+    penaltyStatus = LPPPenaltyStatusEnum.Posted,
+    penaltyAmountAccruing = 123.45,
+    penaltyAmountPosted = 123.45,
+    penaltyChargeCreationDate = LocalDate.of(2022, 1, 1),
+    communicationsDate = LocalDate.of(2022, 1, 1),
+    penaltyChargeReference = "CHARGE123456",
+    penaltyChargeDueDate = LocalDate.of(2022, 2, 1),
+    appealStatus = None,
+    appealLevel = None,
+    principalChargeReference = "CHARGING12345",
+    principalChargeDueDate = LocalDate.of(2022, 3, 1)
+  )
+
+  val lateSubmissionResponseModel: LateSubmission = LateSubmission(
+    lateSubmissionID = "ID123",
+    taxPeriod = Some("1"),
+    taxReturnStatus = "2",
+    taxPeriodStartDate = Some(LocalDate.of(2022, 1, 1)),
+    taxPeriodEndDate = Some(LocalDate.of(2022, 3, 31)),
+    taxPeriodDueDate = Some(LocalDate.of(2022, 5, 7)),
+    returnReceiptDate = Some(LocalDate.of(2022, 4, 1)),
```

Comment:
Can you remove this comma at the end of this parameter list. Even though the compiler is smart enough to ignore this, it throws a scalastyle error which will fail the build. (to run scalastyle on the test package do `sbt test:scalastyle`)